src/FieldEncryptProcessEntities.php

```diff
@@ -0,0 +1,302 @@
+<?php
+/**
+ * @file Contains \Drupal\field_encrypt\FieldEncryptProcessEntities
+ */
+
+namespace Drupal\field_encrypt;
+
+use Drupal\Core\Entity\EntityManager;
+use Drupal\Core\Entity\Query\QueryFactory;
+use Drupal\encrypt\EncryptService;
+
+/**
+ *
+ */
+class FieldEncryptProcessEntities {
+
+  /**
+   * Filter (blacklist) field types that don't work with encryption.
+   *
+   * This list is mostly to avoid problems, but we shouldn't let people encrypt
+   * these fields since we know it wouldn't work.
+   */
+  public $blacklist_fields = [
+    'image',
```

Comment:
@nerdstein, this makes sense to me. All the fields we currently use are text fields. I was using the blacklist to prevent errors and essentially provide a safety net if you accidentally enable to setting on an unsupported field.

I think the plugin system makes sense. I will provide a text plugin with this module and it could be expanded for others. If the plugin is not available, it is skipped.